quinn-proto/src/connection/streams/mod.rs

```diff
@@ -233,6 +233,11 @@ impl<'a> SendStream<'a> {
         self.write_source(&mut BytesArray::from_chunks(data))
     }
 
+    /// Returns the maximum amount of data this is allowed to be written on the connection.
```

Comment:
How is a caller expected to use this? It doesn't seem reasonable to ask them to independently track how many bytes they've written so far.